frontend/src/components/doctors/components/filtration/filtration.tsx

```diff
@@ -3,29 +3,31 @@ import { useForm } from 'react-hook-form';
 import styles from './filtration.module.scss';
 import { TextInput, Checkbox, Details } from 'components/common';
 import {
-  DoctorType,
-  ClinicType,
   InputType,
   InputColor,
   DoctorFiltration,
   Icon,
 } from 'common/enums';
 import { IDoctorFiltrationPayload } from 'common/interfaces';
 import { DEFAULT_FILTER_VALUE } from '../common/constants';
+import { doctorSpecialtiesToReadable, clinicTypesToReadable } from '../common/maps';
 
-const doctorSpecialties = Object.keys(DoctorType);
-const clinicTypes = Object.keys(ClinicType);
+const doctorSpecialties = Object.values(doctorSpecialtiesToReadable);
```

Comment:
```suggestion
const doctorSpecialties = Object.values(doctorSpecialtiesToReadable);
```
Value should be like enum value, label should like in design.
For, example ->
`pediatrician` as value, `Pediatrician` as label